<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiMotionPosition</class>
 <widget class="QWidget" name="UiMotionPosition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_PositionMotion">
       <property name="text">
        <string>Reciprocating Motion</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget_Point">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="4">
        <widget class="QLabel" name="label_PointMotion">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Acceleration:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_PointAcc"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Deceleration:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_PointDec"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Max Velocity:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_PointMaxVel"/>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Pulse:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSpinBox" name="spinBox_PointPulse"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="6">
        <widget class="QLabel" name="label_ReciMotion">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Acceleration:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_ReciAcc"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Deceleration:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_ReciDec"/>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Interval:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_ReciInterval"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Max Velocity:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_ReciMaxVel"/>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Pulse:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSpinBox" name="spinBox_ReciPulse"/>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Times:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QSpinBox" name="spinBox_ReciTimes"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
